src/protocolgame.cpp

```diff
@@ -107,6 +107,22 @@ std::size_t clientLogin(const Player& player)
 	return waitList.size();
 }
 
+ClientDamageType getClientElement(CombatType_t combatType) {
```

Comment:
```suggestion
ClientDamageType getClientDamageType(CombatType_t combatType) {
```
and
`CLIENT_ELEMENTAL_TYPE` to `CLIENT_DAMAGETYPE_TYPE`